docs/developer/solutions/deploy.rst

```diff
@@ -2,24 +2,137 @@ Deploying And Experimenting
 ============================
 
 Given the solution ISO is correctly generated, a script utiliy has been
-added to enable solution install and removal
+added to manage Solutions.
+This script is located at the root of Metalk8s archive:
 
-Installation
-------------
+  .. code::
+
+    /srv/scality/metalk8s-X.X.X/solutions.sh
+
+Import a Solution
+-----------------
+
+Importing a Solution will mount its ISO and expose its container images.
+
+To import a Solution into MetalK8s cluster, use the ``import`` subcommand:
+
+  .. code::
+
+    ./solutions.sh import --archive </path/to/solution.iso>
+
+The ``--archive`` option can be provided multiple times to import several
+Solutions ISOs at the same time:
+
+  .. code::
+
+    ./solutions.sh import --archive </path/to/solution1.iso> \
+      --archive </path/to/solution2.iso>
+
+Unimport a Solution
```

Comment:
Unimport is probably not the right term, I would simply say remove

Reply:
Yes, it's not the right term, it's not even a term. :)
Anyway this stuff will not stay as is and will be changed soon when we'll integrate this script in a centralized CLI.

Reply:
> will be changed soon

`soon` can mean far future or never in IT :stuck_out_tongue:  (except if you mean it'll be tackled in your next PR)

Reply:
Almost my next PR, it's not "maybe, someday... if we have time" :)